packages/cli/__tests__/auth/__system__/__scripts__/auth_login_apiml.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 set -e
 
-zowe auth login apiml
+echo "y" | zowe auth login apiml
```

Comment:
Curious why this command wasn't prompting before but now it is? Or was this a long-standing bug in our system tests? 😢

Reply:
The APIML creates a prompt and asks if the user wants to store their new profile on disk. This seems to be a long standing bug, because otherwise the prompt will continue for about 10 minutes until it times out.